sample/samplers.go

```diff
@@ -90,8 +90,9 @@ func (s *Sampler) sample(tokens []token) (token, error) {
 		sortLogits(tokens)
 	}
 
+	// token logit values are updated to probabilities
```

Comment:
not sure how helpful this comment would be

Reply:
useful to know as topp and minp need probs